src/agent/agent.c

```diff
@@ -584,10 +572,8 @@ bool agent_parse_config(Agent *agent, const char *configfile) {
         }
 
         value = cfg_get_value(agent->config, CFG_MANAGER_PORT);
-        if (value) {
```

Comment:
This checks for `NULL` as well as an empty string as far as I know - this would be missing in `cfg_set_port`.

Reply:
we do the validation in the cfg_set_port